backend_jobs/pipeline_utils/constants.py

```diff
@@ -12,42 +12,6 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 """
-PROJECT_ID = 'projectId'
-
-# Constant collection names:
-IMAGES_COLLECTION_NAME = 'Images'
-PIPELINE_RUNS_COLLECTION_NAME = 'PipelineRuns'
-LABELS_COLLECTION_NAME = 'Labels'
-LABELTAGS_COLLECTION_NAME = 'LabelTags'
-REDEFINE_MAPS_COLLECTION_NAME = 'RedefineMaps'
 
-# Constant field names:
-INGESTED_PROVIDERS = 'ingestedProviders'
-INGESTED_RUNS = 'ingestedRuns'
-PROVIDER_ID = 'providerId'
-PROVIDER_NAME = 'providerName'
-PROVIDER_VERSION = 'providerVersion'
-PROVIDER_VISIBILITY = 'providerVisibility'
-VISIBILITY = 'visibility'
-LABEL_NAME = 'labelName'
-LABEL_IDS = 'labelIds'
-PARENT_IMAGE_ID = 'parentImageId'
-PIPELINE_RUN_ID = 'pipelineRunId'
-HASHMAP = 'hashmap'
-START_DATE = 'startDate'
-END_DATE = 'endDate'
-REDEFINE_MAP = 'redefineMap'
-RANDOM = 'random'
-URL = 'url'
-COORDINATES = 'coordinates'
-DATE_INGESTED = 'dateIngested'
-DATE_SHOT = 'dateShot'
-DATE_FIELDS = 'date'
-IMAGE_ATTRIBUTES = 'imageAttributes'
-FORMAT = 'format'
-RESOLUTION = 'resolution'
-ATTRIBUTION = 'attribution'
-LABELS = 'labels'
-
-INVISIBLE = 0
-VISIBLE = 1
+PROJECT_ID_NAME = 'step-project-ellispis'
+PROJECT_ID = 'projectId'
```

Comment:
this does not seem like something we need to define in constants file. It's a constant parameter of an API - we can inline it directly

Reply:
removed this file and added the constants to the 'backend_jobs/pipeline_utils/firestore_database.py' file which is the onlt place where they are used at.